src/_guides/language/language-tour.md

```diff
@@ -2891,32 +2921,89 @@ class SmartTelevision extends Television {
 }
 {% endprettify %}
 
-Subclasses can override instance methods, getters, and setters. Here’s
-an example of overriding the Object class’s `noSuchMethod()` method,
-which is called whenever code attempts to use a non-existent method or
-instance variable:
 
-<!-- language-tour/no-such-method/bin/main.dart -->
+#### Overriding members
+
+Subclasses can override instance methods, getters, and setters.
+You can use the `@override` annotation to indicate that you are
+intentionally overriding a member:
+
+<!-- language-tour/overrides/bin/override.dart -->
 {% prettify dart %}
-class A {
-  // Unless you override noSuchMethod, using a
-  // non-existent member results in a NoSuchMethodError.
-  void noSuchMethod(Invocation mirror) {
-    print('You tried to use a non-existent member:' +
-          '${mirror.memberName}');
+class SmartTelevision extends Television {
+  @override
+  void turnOn() {
+    // ...
   }
+  // ...
 }
 {% endprettify %}
 
-You can use the `@override` annotation to indicate that you are
-intentionally overriding a member:
+To narrow the type of a method parameter while maintaining type safety,
+use the `covariant` keyword.
```

Comment:
Ah, of course we have to mention `covariant` in this document, because it is indeed a built-in identifier&mdash;so developers _will_ see an error if they try to define a typedef or a class with that name, so they do need to know about it.

But we only need to say that it has no effect, and that it only exists due to compatibility with strong mode.